models/ed/R/veg2model.ED2.R

```diff
@@ -0,0 +1,234 @@
+#' Writes ED specific IC files
+#'
+#' @param obs data either passed from upstram or NULL
+#' @param inputinfo
+#' @param runinfo run from settings
+#' @param outfolder where to write files
+#' @param host 
+#' @param con bety$con
+#' @param overwrite
+#' @return ic.id initial conditions file input id
+#' @export
+#' @author Istem Fer
+#'  We need the in.path, the out folder, the start and endates and the lat/long. All of these are in setting$run. 
+veg2model.ED2 <- function(obs = NULL, inputinfo, runinfo, outfolder, host, con, overwrite = FALSE){
+  
+  start_year <- lubridate::year(runinfo$start.date)
```

Comment:
I believe we're trying to move away from the  'package::function' method of calling external functions to using the '@importFrom package function1 function2' method in the roxygen heading so that it can include these imports in the NAMESPACE.